src/pages/index.js

```diff
@@ -111,3 +197,48 @@ profileAddCardButton.addEventListener("click", () => {
   addFormValidator.resetValidation();
 });
 
+profileAvatarButton.addEventListener("click", () => {
+  avatarEditPopup.open();
+});
+
+const confirmDelete = new PopupWithConfirmation("#confirm-delete-modal");
+confirmDelete.setEventListeners();
+
+
+function handleDeleteClick(card) {
+  confirmDelete.open();
+  confirmDelete.setSubmitAction(() => {  
+    api
+    .deleteCard(card._id)
+    .then(() => {
+      card.handleDeleteClick();
+      confirmDelete.close();
+    })
+    .catch((err) => {
+      console.error(err);

```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors